README.md

```diff
@@ -340,6 +340,12 @@ Again, with all these methods, You will want your application to link to an inte
 > NOTE:
 > A JWT will expire after 60 seconds of creation.
 
+#### Exchange ID Site token for a Stormpath Access Token
+After the user has been authenticated via ID Site, a developer may want to control their authorization with an OAuth 2.0 Token. 
+This is done by passing the JWT similar to the way we passed the user’s credentials as described in [Generating an OAuth 2.0 Access Token][generate-oauth-access-token]. 
+The difference is that instead of using the password grant type and passing credentials, we will use the id_site_token type and pass the JWT we got from the ID Site
+more info [here][exchange-id-site-token].
```

Comment:
I think we will need an example of how this works in Ruby. Just how the calls would look like for the user.